index.bs

```diff
@@ -4377,6 +4378,38 @@ navigated</dfn> steps given |context| and |navigation status|:
 
 </div>
 
+#### The browsingContext.historyUpdated Event #### {#event-browsingContext-historyUpdated}
+
+<dl>
+   <dt>Event Type</dt>
+   <dd>
+      <pre class="cddl local-cddl">
+        browsingContext.HistoryUpdated = (
+         method: "browsingContext.historyUpdated",
+         params: {
+            url: text,
+         }
+        )
+      </pre>
+   </dd>
+</dl>
+
+<div algorithm>
+The [=remote end event trigger=] is the <dfn export>WebDriver BiDi history updated</dfn> steps given |navigable|:
```

Comment:
Navigables renaming is not required since this is for html spec that is already written in terms of navigables. Navigation status does not apply here since history update is not a navigation.

Reply:
Fragment navigated is a navigation:

1) fragment navigations go through the navigate algorithm https://html.spec.whatwg.org/multipage/browsing-the-web.html#navigate-fragid

2) non fragment “navigations” (in quotes) do not go through the navigate algorithm https://html.spec.whatwg.org/#navigate-non-frag-sync

Reply:
After a discussion, we changed the PR to emit the event in https://html.spec.whatwg.org/#shared-history-push/replace-state-steps as it seems to better match the use case. The previously considered integration point in `URL and history update steps` seems to be used internally by the HTML spec for regular navigation types.